tracer/src/Datadog.Trace/PlatformHelpers/AspNetCoreHttpRequestHandler.cs

```diff
@@ -98,7 +98,7 @@ public Scope StartAspNetCorePipelineScope(Tracer tracer, HttpContext httpContext
         {
             string host = request.Host.Value;
             string httpMethod = request.Method?.ToUpperInvariant() ?? "UNKNOWN";
-            string url = request.GetUrl(tracer.TracerManager.QueryStringManager);
+            var url = request.GetUrl(tracer.TracerManager.QueryStringManager);
```

Comment:
style nit-picking: why change this one and leave the ones above? I'd rather be consistent. Either make them all `string` or all `var`.